src/senaite/core/browser/static/js/senaite.core.analysisrequest.add.coffee

```diff
@@ -458,7 +458,7 @@ class window.AnalysisRequestAdd
     base_url = window.location.href
     if base_url.search("/portal_factory") >= 0
       return base_url.split("/portal_factory")[0]
-    return base_url.split("/ar_add")[0]
+    return document.body.dataset.baseUrl
```

Comment:
I think we can remove the complete upper part as well, so that this method simply returns `document.body.dataset.baseUrl`.
If I remember correctly, this `portal_factory` dance was done before we programmatically created a temporary object in code to gather all the standard/extended fields